manifests/d/DevToys-app/DevToys/2.0-preview.3/DevToys-app.DevToys.locale.en-US.yaml

```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: DevToys-app.DevToys
+PackageName: DevToys
+PackageVersion: 2.0-preview.3
+PackageLocale: en-US
+Publisher: DevToys-app
```

Comment:
Windows ARP (settings > apps > installed apps) shows `DevToys` not `DevToys-app`